irohad/util/shepherd.cpp

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <iostream>
+
+#include "util/proto_status_tools.hpp"
+
+#include <gflags/gflags.h>
+#include "common/irohad_version.hpp"
+#include "logger/logger.hpp"
+#include "logger/logger_manager.hpp"
+#include "main/iroha_conf_literals.hpp"
+#include "util/utility_client.hpp"
+
+static bool validateVerbosity(const char *flagname, const std::string &val) {
+  const auto it = config_members::LogLevels.find(val);
+  if (it == config_members::LogLevels.end()) {
+    std::cerr << "Invalid value for " << flagname << ": should be one of ";
+    for (const auto &level : config_members::LogLevels) {
+      std::cerr << " '" << level.first << "'";
+    }
+    std::cerr << "." << std::endl;
+    return false;
+  }
+  return true;
+}
+
+static bool validateSingleAction(const char *flagname, bool val) {
+  static bool got_a_command = false;
+  if (got_a_command && val) {
+    std::cerr << "More than one command specified!";
+    return false;
+  }
+  got_a_command |= val;
+  return true;
+}
```

Comment:
'flagname' what for?

Reply:
it is the interface of gflags lib: https://gflags.github.io/gflags/#validate

Reply:
Than we need to 'static bool validateSingleAction(const char */*flagname*/, bool val)' to remove warning of unused variable.

Reply:
done